_includes/pbc/all/install-features/202407.0/marketplace/install-the-merchant-portal-feature.md

```diff
@@ -355,7 +359,7 @@ composer require spryker/security-gui:"^1.0.0" spryker/security-merchant-portal-
 | User         | vendor/spryker/user          |
 | ZedUi        | vendor/spryker/zed-ui        |
 
-3. Apply changes from https://github.com/spryker-shop/suite/pull/681/files.
+3. Apply changes from https://github.com/spryker-shop/suite/pull/681/files to setup a separate login for Merchant Portal user.
```

Comment:
This says the same as the title of the section. What changes does this PR contain?